Makefile

```diff
@@ -9,6 +9,8 @@ BUILD_GOPATH=$(TARGET_DIR)
 
 #inputs to 'run' which may need to change
 TLS_CERTS_BASEDIR=_output
+NAMESPACE ?= "cluster-logging"
```

Comment:
Where is this used?  If we really need it, likely should be `openshift-logging`, the default cluster-logging namespace

Reply:
I will change the default.

Elasticsearch can be deployed in any ns. In Jaeger we deploy it any really any namespace.